src/components/view/ReservationForm.js

```diff
@@ -0,0 +1,112 @@
+import React, { useState, useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useParams, useNavigate } from 'react-router-dom';
+import { createReservation } from '../../redux/ReservationsSlice';
+import { fetchLocations } from '../../redux/LocationsSlice';
+import { fetchServices } from '../../redux/ServicesSlice';
```

Comment:
- [ ] Kindly Update the `fetchServices` to `fetchMovies` since in the `dev` branch there exists  `fetchMovies` redux thunk.